File: src/ejerciciosHerencia/Vehiculo.java
package ejerciciosHerencia;
/**
 * Clase Vehiculo
 * @author d18momoa
 *
 */
public class Vehiculo {
	
	private static int kilometrosTotales = 0;
	private static int vehiculosCreados = 0;
	private int kilometrosRecorridos = 0;

	public Vehiculo(int kmR) {
		this.kilometrosRecorridos = kmR;
    kilometrosTotales += kmR;
    vehiculosCreados += 1;
	}
	private void setKilometrosRecorridos(int kilometrosRecorridos) {
		this.kilometrosRecorridos = kilometrosRecorridos;
	}
	public int getKilometrosRecorridos() {
		return kilometrosRecorridos;
	}
	public int getKilometrosTotales() {
		return kilometrosTotales;
	}
	public int getVehiculosCreados() {
		return vehiculosCreados;
	}
	public void echarAAndar() {
		System.out.println("En marcha");
	}
}    

File: ejerciciosHerencia/Vehiculo.py
'''
Created on 27 ene. 2019
Clase Vehiculo
@author: d18momoa
'''

class Vehiculo():
    
    kilometrosTotales = 0
    vehiculosCreados = 0
    
    def __init__(self,kmR):
        self.__kilometrosRecorridos = kmR
        self.kilometrosTotales += kmR
        self.vehiculosCreados += 1
    def getKmTotales(self):
        return self.kilometrosTotales
    def getVehiculosCreados(self):
        return self.vehiculosCreados
    def getKmRecorridos(self):
        return self.__kilometrosRecorridos
    def echarAAndar(self):
        print("En marcha!")  